heads/src/main/java/com/dumbdogdiner/stickyapi/bukkit/item/SkullBuilder.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bukkit.item;
+
+import com.dumbdogdiner.stickyapi.common.util.StringUtil;
+import com.dumbdogdiner.stickyapi.util.textures.TextureHelper;
+import com.dumbdogdiner.stickyapi.util.textures.TextureValidator;
+import com.google.common.base.Preconditions;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.VisibleForTesting;
+
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.SkullMeta;
+import com.destroystokyo.paper.profile.PlayerProfile;
+import com.destroystokyo.paper.profile.ProfileProperty;
+
+import java.net.URL;
+import java.util.UUID;
+
+/**
+ * Utility for constructing ItemStacks of {@link Material#PLAYER_HEAD}.
+ * <p>
+ * You can specify a group and name from the embedded texture file, or simply set a name and a texture,
+ * and generate player heads with ease!
+ * </p>
+ */
+
+@SuppressWarnings("UnusedReturnValue")
+public class SkullBuilder {
+    @Getter
+    @NotNull
+    @VisibleForTesting
+    private String category = "*";
```

Comment:
IIRC lombok will translate javadoc onto the generated methods - worth doing?

Reply:
It just gets the category, I can write that but it's obvious imo

Reply:
Yes, because you wrote the code. Please document things :3

Reply:
More specifically - what IS category?

Reply:
The first part of the YML (Player, block, mob, mhf, misc, etc)

Reply:
Okay, just be sure to leave a comment that says that!

Reply:
Bump